lib/_http_outgoing.js

```diff
@@ -92,7 +92,7 @@ OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
 
   if (callback) {
     if (typeof callback !== 'function')
-      throw new TypeError('callback must be a function');
+      throw new TypeError('"callback" must be a function');
```

Comment:
Can we include a period at the end of all messages? They are full sentences after all.